tket-exts/src/tket_exts/tket/bool.py

```diff
@@ -0,0 +1,78 @@
+"""Opaque boolean operations."""
+
+import functools
+from typing import List
+
+from hugr.ext import Extension, OpDef, TypeDef
+from hugr.ops import ExtOp
+from hugr.tys import ExtType
+from ._util import TketExtension, load_extension
+
+
+class BoolExtension(TketExtension):
+    """Opaque boolean extension.
+
+    This extension operates with an opaque `bool_t` type that
+    is equivalent to Hugr's `[] + []` sum type representation.
+    """
+
+    @functools.cache
+    def __call__(self) -> Extension:
+        """Return the bool extension"""
+        return load_extension("tket.bool")
+
+    def TYPES(self) -> List[TypeDef]:
+        """Return the types defined by this extension"""
+        return [self.bool_t.type_def]
+
+    def OPS(self) -> List[OpDef]:
+        """Return the operations defined by this extension"""
+        return [
+            self.and_op.op_def(),
```

Comment:
nit: This isn't consistent with the other extensions, which define methods do get the OpDef and methods for the ExtOp